website/content/docs/pipelines/index.mdx

```diff
@@ -0,0 +1,300 @@
+---
+layout: docs
+page_title: Pipelines
+description: |-
+  The overview of Waypoint Pipelines.
+---
+
+# Waypoint Custom Pipelines
+
+~> **NOTE**: Custom Pipelines is currently a **Tech Preview** feature of Waypoint.
+Given that, we invite all Waypoint users to give custom pipelines a try and let
+us know your experience with it, feature requests and enhancements, or any bugs
+you might find via our [Discuss forums](https://discuss.hashicorp.com/c/waypoint/51)
+or [GitHub](https://github.com/hashicorp/waypoint/issues). We look forward to
+hearing about your experience and ways Waypoint can help make your experience
+delivering your applications even better!
```

Comment:
💨 make it clear which things go to Discuss and which go to GitHub, and don't say `experience` twice in the same sentences
```suggestion
Given that, we invite all Waypoint users to give custom pipelines a try and let
us know your experience with it on our [Discuss forums](https://discuss.hashicorp.com/c/waypoint/51),
and feature requests, enhancements, or any bugs you might find 
via [GitHub](https://github.com/hashicorp/waypoint/issues). We look forward to
hearing what you think and ways Waypoint can help make your experience
delivering your applications even better!
```